### ISSUE TITLE
Missing aria for the "stop" button in screen reader

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-AssemblyLine#315]
T&P 25-1

**Comment by plocket** [comment]
I think this may be, in practice, the same issue as #336. Trying to find the specific description of this item.

**Comment by plocket** [comment]
Just found it and it looks like the same issue.
